guides/common/modules/ref_troubleshooting-errors.adoc

```diff
@@ -0,0 +1,32 @@
+[id="Troubleshooting_Errors_{context}"]
+= Troubleshooting Errors
+
+== "[Errno 1] Operation not permitted: ..." during repository syncing:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# chown --recursive pulp.pulp /var/lib/pulp/media/
+----
+
+== "{“policy”:[""" is not a valid choice."]}" during Debian repository syncing:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-rake katello:migrate_deb_content_attributes_to_pulp3
+----
+
+== 500 API error during syncing with "cryptography.fernet.InvalidToken" in /var/log/messages traceback:
```

Comment:
```suggestion
500 API error during syncing with "cryptography.fernet.InvalidToken" in /var/log/messages traceback::
```

Reply:
Apologies for this as my `make BUILD` is broken now after upgrading to F38. So, cannot build locally as of now.

Addressed :+1: